.claude/skills/bt-test/SKILL.md

```
@@ -20,28 +20,30 @@ Run the test suite with various options.
-   - `coverage`: Run `pnpm run test:unit:coverage` (with coverage thresholds)
-   - `watch`: Run `pnpm run test:unit:watch` (interactive watch mode)
-   - `visual`: Run `pnpm run test:visual` (Playwright visual regression)
-   - File path: Run `pnpm exec vitest run <path>`
+
+- No arguments: Run `pnpm run test:unit` (all Vitest tests)
+- `coverage`: Run `pnpm run test:unit:coverage` (with coverage thresholds)
+- `watch`: Run `pnpm run test:unit:watch` (interactive watch mode)
+- `visual`: Run `pnpm run test:visual` (Playwright visual regression)
+- File path: Run `pnpm exec vitest run <path>`
 
 2. **Report results**
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)